### DISCUSSION TITLE
<title> How to copy and paste formatted text into the wiki?

### DISCUSSION THREAD

**Discussion by ReadySetGooo** [community/community#69543] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi. I have text I want to copy and paste into the wiki. The text has formatting like bold, bullet points, etc. When I try copy and pasting, none of the formatting transfers. How can I do it?

**Comment by Vijaykv5** [comment]
Hi @ReadySetGooo  

To preserve text formatting when copying and pasting into a wiki, follow these steps: 
1. Understand the wiki's markup language. 
2. Paste the text into a plain text editor.
3.  Manually apply formatting using the wiki's markup (e.g., double apostrophes for bold, asterisks for bullet points).
4. Paste the formatted text into the wiki's editor. 
5. Preview and adjust formatting as needed.
6. Save the page. 
Be sure to consult the specific wiki's documentation for its markup rules. This process ensures your text retains its formatting when transferred to the wiki, helping maintain readability and structure. Hope it helps.

**Reply by ReadySetGooo** [reply]
thank you! I think I might just edit it the normal way then.
